float32 leaf_size
string save_path
---
bool success
